ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseWfsLayers/BrowseWfsLayers.cpp

```diff
@@ -146,12 +147,11 @@ void BrowseWfsLayers::populateWfsFeatureTable()
   // query the service
   constexpr bool clearCache = false;
   const QStringList outFields {"*"};
-  connect(m_wfsFeatureTable, &WfsFeatureTable::populateFromServiceCompleted, this, [this]()
+  m_wfsFeatureTable->populateFromServiceAsync(params, clearCache, outFields).then(this, [this](const FeatureQueryResult*)
   {
     addFeatureLayerToMap();
```

Comment:
You should delete the FeatureQueryResult* in the handler since you're not using it. That is one of our promoted patterns in the samples. You can stick it in a std::unique_ptr.

It won't leak like this, but we should clean up memory in async handlers like this when possible and we're done with the result.

Reply:
Used std::unique_ptr.